### PR TITLE
Use Toggle Task for deployment of cassandra exporter, bump to Kubernetes 1.16

### PR DESCRIPTION
**Pull request by ANeumann82**
Signed-off-by: Andreas Neumann <aneumann@mesosphere.com>

### CODE REVIEW

**Comment by ANeumann82** [comment]
Waiting for merge and release of https://github.com/kudobuilder/kudo/pull/1552 before we can merge this

File: metadata.sh [hunk 1]
```diff
@@ -40,9 +40,9 @@ export VENDOR_DIRECTORY="${_project_directory}/shared/vendor"
 export CASSANDRA_VERSION="3.11.6"
 
 # https://github.com/kudobuilder/kudo/releases/tag/vx.y.z
-export KUDO_VERSION="0.13.0"
+export KUDO_VERSION="0.15.0-rc1"
 
-export KUBERNETES_VERSION="1.15.0"
+export KUBERNETES_VERSION="1.16.0"
```
Comment (porridge) [inline metadata.sh:46]: Please mention these bumps in commit message and/or title.

File: operator/params.yaml [hunk 2]
```diff
@@ -227,6 +227,7 @@ parameters:
   ################################################################################
 
   - name: PROMETHEUS_EXPORTER_ENABLED
+    description: "A toggle to enable the prometheus metrics exporter"
```
Comment (porridge) [inline operator/params.yaml:230]:
```suggestion
    description: "A toggle to enable the prometheus metrics exporter."
```
For consistency.

File: operator/templates/service-monitor.yaml [hunk 3]
```diff
@@ -17,5 +16,4 @@ spec:
   selector:
     matchLabels:
       kudo.dev/instance: {{ .Name }}
-      kudo.dev/servicemonitor: "true"
-{{ end }}
+      kudo.dev/servicemonitor: "true"
```
Comment (porridge) [inline operator/templates/service-monitor.yaml:19]: Can you please keep the trailing newline? It's visual noise when it's getting added and removed.

File: templates/operator/params.yaml.template [hunk 4]
```diff
@@ -227,6 +227,7 @@ parameters:
   ################################################################################
 
   - name: PROMETHEUS_EXPORTER_ENABLED
+    description: "A toggle to enable the prometheus metrics exporter"
```
Comment (porridge) [inline templates/operator/params.yaml.template:230]:
```suggestion
    description: "A toggle to enable the prometheus metrics exporter."
```

File: operator/templates/service-monitor.yaml (hunk 3 above)
Comment (ANeumann82) [reply operator/templates/service-monitor.yaml:19]: Well, the change here was the removal of `{{ end }}` in the last line, not the trailing new line, but i've added it nonetheless back in.

**Pull request merged**